src/xrEngine/x_ray.cpp

```diff
@@ -393,7 +393,11 @@ void CApplication::Level_Scan()
     Levels.clear();
 
     xr_vector<char*>* folder = FS.file_list_open("$game_levels$", FS_ListFolders | FS_RootOnly);
-    //. R_ASSERT (folder&&folder->size());
+    if (!folder)
+    {
+        Msg("! No levels found in game data");
```

Comment:
There's no need to call Msg if we don't need to format log.
```suggestion
        Log("! No levels found in game data");
```